src/adapters/WsEasyRtcAdapter.js

```diff
@@ -5,9 +5,14 @@ class WsEasyRtcInterface {
     this.app = 'default';
     this.room = 'default';
     this.connectedClients = [];
+
+    this.serverTimeRequests = 0;
```

Comment:
There's starting to be a lot of repeated code between this and EasyRtcAdapter.js. We can worry about this in a separate PR but I think it'd be useful to change back to having a `webrtc` flag in `networked-scene` that you can toggle.

Reply:
[I've created an issue for this](https://github.com/networked-aframe/networked-aframe/issues/80)